doc/source/workflow/overview.md

```diff
@@ -0,0 +1,208 @@
+4. Deployment manifest (`SeldonDeployment` CRD) is created and applied to the Kubernetes cluster.
+5. Seldon Core `Operator` creates all required Kubernetes resources.
+6. Inference requests are served by Seldon Core `Orchestrator`.
+7. Metrics and tracing data is written to the third party systems.
+
+## Model Servers
+
+With Seldon Core you can build two type of servers: reusable and non-reusable ones.
+Each of these are useful depending on the context and the actual use case.
+
+- **Reusable Model Servers**: Often referred to as prepackaged model servers.
+  Allow to deploy a family of similar models without the need to build a new server each time.
+  They often fetch models from a central repository (like your company's S3 storage)
+- **Non-Reusable Model Servers**: Specialised server meant to serve a single model.
+  Does not require the central repository but requires a build of a new image for every model.
+
+
+![](../images/model-servers.svg)
+
+Read more about our pre-packaged `Model Servers` on their dedicated documentation pages:
+- [MLflow Server](../servers/mlflow.html)
+- [SKLearn Server](../servers/sklearn.html)
+- [Tensorflow Server](../servers/tensorflow.html)
+- [XGBoost Server](../servers/xgboost.html)
+
+Read how to build your own pre-packaged model server [here](../servers/custom.html).
+
+## Language Wrappers
+
+Language wrappers allows Seldon Core users to build reusable and non-reusable model servers.
+As you will see, the whole process is very simple and requires user to only define logic that
+loads models and perform inference prediction as well as required runtime dependencies.
+
+![](../images/language-wrappers-1.svg)
+
+
+Model loading and inference logic is defined in `Model.py` file:
+```python
+class Model:
+  def __init__(self):
+    # custom logic that prepare model
+    self._model = ...
+
+  def predict(self, features, names=[], meta=[]):
+    # custom inference logic
+    return self._model.predict(...)
+```
+
+If `Model`'s `__init__` method does not contain logic to load models this would be the non-reusable model server.
+
+The `seldon-core-microservice` Python wrapper can be used to turn `Model.py` into a fully operational microservice.
+
+![](../images/language-wrappers-2.svg)
+
+To complete containerisation process you need two more components:
+- `requirements.txt` file that describes your runtime dependencies
+- `.s2/environment` file that describes your microservice (api and model type)
+
+Once these are in place you can use a simple s2i command
+```bash
+s2i build . seldonio/seldon-core-s2i-python3:1.1.0 model:0.1
+```
+to create ready to use Docker image.
+
+Read more about Python `Language Wrapper` on its dedicated documentation [page](../python/index.html).
+
+
+## Seldon Deployment CRD
+
+Seldon Deployment CRD is the real strength of Seldon Core. It allows you to easily deploy your inference model to the Kubernetes cluster and handle some real production traffic!
```

Comment:
It may be worth explaining what a CRD is, and how it provides abstractions to native kubernetes deployments / pods specialised for ML

Reply:
This could actually be a good opportunity to talk about the graph capabilities of seldon core, or at least more functional components it brings (as right now it mostly just emphasises that its great)

Reply:
I extended this section in updated version (push in a moment)